app/forms/appropriate_body_selection_form.rb

```diff
@@ -15,14 +15,14 @@ class AppropriateBodySelectionForm
 
   validates :body_appointed,
             inclusion: { in: %w[yes no],
-                         message: "Please select whether you have appointed an appropriate body or not" },
+                         message: "Select whether you have appointed an appropriate body or not" },
```

Comment:
```suggestion
                         message: "Select whether you’ve appointed an appropriate body or not" },
```

Reply:
Thanks for removing the 'please' instances. I've added extra amendments to bring all content in-line with the GDS style guide.